docs/rebuild_admin.rst

```diff
@@ -195,7 +189,7 @@ First, log on to the *Application Server* via the console and edit the file
 
 Then, restart ``sshd`` using the command ``sudo service sshd restart``.
 
-Finally, check the file ``/etc/network/interfaces``, and note down the network
+Finally, check the file ``/etc/netplan/10-ifupdown.yaml``, and note down the network
```

Comment:
That file does not exist on my 20.04 app server. I have an `/etc/netplan` directory, and it contains `00-installer-config.yaml`, but no `10-ifupdown.yaml`.

Reply:
Right. `10-ifupdown.yaml` is for those guilty of `netplan migrate`, but not clean install, and that's the last state my servers were in. Shall update.